ocs_ci/helpers/helpers.py

```diff
@@ -538,6 +539,12 @@ def create_storage_class(
         )
         if rbd_thick_provision:
             sc_data["parameters"]["thickProvision"] = "true"
+        if encrypted:
+            # Avoid circular imports
+            from ocs_ci.utility.kms import get_encryption_kmsid
+
+            sc_data["parameters"]["encrypted"] = "true"
+            sc_data["parameters"]["encryptionKMSID"] = get_encryption_kmsid()
```

Comment:
I know we discussed about it quite a bit offline, but I have another suggestion, since there can be multiple KMS IDs present in the configmap and how they are used can vary depending on the test case, can we have encryptionKMSID passed as a parameter while creating the SC, so that we can use it accordingly, based on what the test case requirements are?

Reply:
@rachael-george , I had confusions in encryptionKMSID on how it gets generated with new id numbers. 
so https://github.com/red-hat-storage/ocs-ci/pull/4463/files#diff-5c36a978fe9387d4958d42a61c55682c2f9abdbcd4546cb8e25371a0ae017ee9R736 here I have hardcoded the id number.

Can we merge this and immediately create another PR which handles just the encryptionKMSID parameter ?

Reply:
Sure, we can do that.